analytics-webapps/src/main/webapp/WEB-INF/conf/analytics/portal-configuration.xml

```diff
@@ -72,7 +72,7 @@
           <name>category</name>
           <object type="org.exoplatform.webui.core.model.SelectItemCategory">
             <field name="name">
-              <string>analyticsPageConfigs</string>
+              <string>normalPageConfigs</string>
```

Comment:
why changing the name of Page template knowing that it will reference the Analytics Page Template https://github.com/Meeds-io/analytics/blob/8a2a1bbf6dfff65a1b6aed308ce0d2c7d4935f82/analytics-webapps/src/main/webapp/WEB-INF/conf/portal/template/pages/analytics/page.xml that hasn't changed. Besides, by changing this name, the I18N  isn't applied any more : 
https://github.com/Meeds-io/analytics/blob/8a2a1bbf6dfff65a1b6aed308ce0d2c7d4935f82/analytics-webapps/src/main/resources/locale/portal/webui_en.properties#L1

Reply:
@boubaker We have changed just the category in which analytics page template should be. This is what is already excepted in the mip37: 1.1 Remove current pages templates (Priority 1)